web/src/features/moreCast2/components/GridComponentRenderer.tsx

```diff
@@ -44,13 +44,14 @@ export class GridComponentRenderer {
     precision: number,
     field: string
   ): string => {
-    const actualField = this.getActualField(field)
-    const actual = params.row[actualField]
+    if (field.includes('grass')) {
```

Comment:
Don't necessarily love this....it's because I went the Actual/Forecast route in handling GC values. Maybe should've gone a different direction with that